src/include/daos/dtx.h

```diff
@@ -161,14 +161,15 @@ daos_dti_equal(struct dtx_id *dti0, struct dtx_id *dti1)
 #define DP_DTI(dti)	DP_UUID((dti)->dti_uuid), (dti)->dti_hlc
 
 enum daos_ops_intent {
-	DAOS_INTENT_DEFAULT	= 0,	/* fetch/enumerate/query */
-	DAOS_INTENT_PURGE	= 1,	/* purge/aggregation */
-	DAOS_INTENT_UPDATE	= 2,	/* write/insert */
-	DAOS_INTENT_PUNCH	= 3,	/* punch/delete */
-	DAOS_INTENT_REBUILD	= 4,	/* for rebuild related scan */
-	DAOS_INTENT_CHECK	= 5,	/* check aborted or not */
-	DAOS_INTENT_KILL	= 6,	/* delete object/key */
-	DAOS_INTENT_COS		= 7,	/* add something into CoS cache. */
+	DAOS_INTENT_DEFAULT		= 0, /* fetch/enumerate/query */
+	DAOS_INTENT_PURGE		= 1, /* purge/aggregation */
+	DAOS_INTENT_UPDATE		= 2, /* write/insert */
+	DAOS_INTENT_PUNCH		= 3, /* punch/delete */
+	DAOS_INTENT_REBUILD		= 4, /* for rebuild related scan */
+	DAOS_INTENT_CHECK		= 5, /* check aborted or not */
+	DAOS_INTENT_KILL		= 6, /* delete object/key */
+	DAOS_INTENT_COS			= 7, /* add something into CoS cache. */
+	DAOS_INTENT_IGNORE_NONCOMMITTED	= 8, /* ignore non-committed DTX. */
```

Comment:
Nit-picking: "active" seems to be a more conventional term to refer to a transaction that has began but not yet committed/aborted.

Reply:
OK, will rename it in subsequent patch.